source/slang/slang-diagnostic-defs.h

```diff
@@ -753,7 +753,7 @@ DIAGNOSTIC(41201, Warning, expectDynamicUniformValue, "value stored at this loca
 DIAGNOSTIC(41202, Error, notEqualBitCastSize, "invalid to bit_cast differently sized types: '$0' with size '$1' casted into '$2' with size '$3'")
 DIAGNOSTIC(41203, Warning, notEqualReinterpretCastSize, "reinterpret<> into not equally sized types: '$0' with size '$1' casted into '$2' with size '$3'")
 
-DIAGNOSTIC(41300, Error, byteAddressBufferUnaligned, "Base offset '$0' of the byteAddressBuffer access is not divisible by the explicitly defined alignment val `$1`")
+DIAGNOSTIC(41300, Error, byteAddressBufferUnaligned, "Unknown Base offset prevents us from performing an aligned byteAddressBuffer access since the explicit alignment '$0' is not divisible by the size of the base type `$1`")
```

Comment:
I don't understand this error message. Why is `offset` unknown? We generate the error because it is known.

Reply:
I meant to say, the value IRInst* buffer can be equated to a IntLiteral and requires the user to specify alignment explicitly.
I modified the error message to this. Any better?
```
invalid alignment `$0` specified for the byte address buffer resource with the element size of `$1`
```